Tests/RenderingTests/Modifiers/NavigationModifiersTests.swift

```diff
@@ -15,18 +15,18 @@ final class NavigationModifiersTests: XCTestCase {
         #if os(iOS)
         try assertMatch(
             #"""
-            <TabView modifiers="[{&quot;style&quot;:&quot;page_always&quot;,&quot;type&quot;:&quot;tab_view_style&quot;}]">
-                <Text modifiers="[{&quot;label&quot;:&quot;label&quot;,&quot;type&quot;:&quot;tab_item&quot;}]">
+            <TabView modifiers='[{"style":"page_always","type":"tab_view_style"}]'>
+                <Text modifiers='[{"label":"label","type":"tab_item"}]'>
                     A
                     <Image template="label" system-name="person.crop.circle.fill" />
                 </Text>
-                <Text modifiers="[{&quot;label&quot;:&quot;label&quot;,&quot;type&quot;:&quot;tab_item&quot;}]">
+                <Text modifiers='[{"label":"label","type":"tab_item"}]'>
                     B
                     <Image template="label" system-name="tray.fill" />
                 </Text>
             </TabView>
             """#,
-            size: .init(width: 100, height: 100)
+            size: .init(width: 200, height: 200)
```

Comment:
This one was weird. The test failed when using `drawingGroup`. The images looked the same to my eyes but the tab items were stacked on top of each other because I presume the container wasn't large enough. Increasing the size allowed it to pass.